### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by FinalAngel**
### Summary

This PR improves on GitHub issue creation and Contribution guidelines.

### General checklist

* [x] I have updated the CHANGELOG.txt

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/---bug-report.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ assignees: ''
 ---
 
 <!--
-Please fill out each section below, otherwise, your issue will be closed. This info allows django CMS maintainers to diagnose (and fix!) your issue as quickly as possible.
+Please fill out each section below, otherwise, your issue will be closed.
```
Comment (evildmp) [inline .github/ISSUE_TEMPLATE/---bug-report.md]: Fill *in* rather than out. Or simply, *complete*.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,23 +1,25 @@
+## Description
+
 <!--
-    If this is a security-related patch stop immediately!
-    See http://docs.django-cms.org/en/latest/contributing/development-policies.html
+If this is a security issue stop right here and follow our documentation:
```
Comment (evildmp) [inline .github/PULL_REQUEST_TEMPLATE.md]: ``... stop immediately and follow the instructions at http://docs.django-cms.org/en/latest/contributing/development-policies.html#reporting-security-issues.``

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,23 +1,25 @@
...
-### General checklist
+<!--
+Please check the following items before submitting, otherwise, your pull request will be closed.
+-->
 
-* [ ] I have updated the CHANGELOG.txt
-* [ ] I have created backports if necessary
-* [ ] I have updated the documentation and/or amended the upgrade section
-      if necessary
+* [x] I have opened this pull request against ``develop``
```
Comment (evildmp) [inline .github/PULL_REQUEST_TEMPLATE.md]: Why is this one pre-checked?

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -0,0 +1,71 @@
+============
+CONTRIBUTING
+============
+
+First, thank you very much, that you would like to contribute to django CMS. 🎉
```
Comment (evildmp) [inline CONTRIBUTING.rst]: Eugh more emojicons.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (FinalAngel) [reply .github/PULL_REQUEST_TEMPLATE.md]: to indicate how to check something, there were several issues where it was either [* ], [check] or [. ]

**Pull request merged**